amethyst_core/src/transform/systems.rs

```diff
@@ -169,17 +185,29 @@ impl<'a> System<'a> for TransformSystem {
             index += 1;
         }
 
-        self.dirty.clear();
+        println!("with parent updated: {:?}", parented);
+        println!("no parent updated: {:?}", noparents);
+
+        (&mut locals).open().1.clear_flags();
+        (&mut parents).open().1.clear_flags();
+
+        for bit in (&self.frame_init).iter() {
```

Comment:
No need for `.iter()` I think

Reply:
BitSetLikes don't implement IntoIterator, but they probably should so: https://github.com/slide-rs/hibitset/pull/18